src/main/java/uk/gov/hmcts/reform/civil/launchdarkly/FeatureToggleService.java

```diff
@@ -31,11 +31,6 @@ public boolean isFeatureEnabled(String feature, LDUser user) {
         return internalClient.boolVariation(feature, user, false);
     }
 
-    public boolean isOrganisationOnboarded(String orgId) {
-        LDUser ldUser = createLDUser().custom("orgId", orgId).build();
-        return internalClient.boolVariation("isOrganisationOnboarded", ldUser, false);
-    }
-
```

Comment:
Not sure about this. Is it related to the feature toggle?

Reply:
From the ticket:
Acceptance Criteria
AC1 - Remove isOrganisationOnboarded toggle from FeatureToggleService and any other classes where it might be referenced and launch darkly.

Reply:
Nevermind. It's in the ticket's acceptance criteria.